src/modules/developer-guide/pages/server-plugins/index.adoc

```diff
@@ -0,0 +1,145 @@
+include::ROOT:partial$variables.adoc[]
+
+= Server plugins
+
+{PRODUCT_NAME} server can be extended using https://pkg.go.dev/plugin[Go plugins].
```

Comment:
{PRODUCT_NAME} implements the support for https://pkg.go.dev/plugin[Go plugins] which can be used to...

IMO rewriting it like so makes it sound nicer.